### ISSUE TITLE
Mob Rendering is Causing my World to crash

### ISSUE THREAD

**Issue by TronWoWGamer** [teamfusion/rottencreatures#23]
I was flying over the desert when a certain mob/armor rendering caused my game to crash and I can't log back in.  I am uploading the crash file.  Not certain the error is with the mob rendering or the armor rendering or the naming convention of the armor.

[crash-2023-04-25_16.25.52-client.txt](https://github.com/teamfusion/rottencreatures/files/11326823/crash-2023-04-25_16.25.52-client.txt)

**Comment by TronWoWGamer** [comment]
I got around it temporarily by editing my location with NBT Explorer and sent myself back home so it would stop trying to spawn.  Staying away from that area for now, lol.

**Comment by Gbergz** [comment]
Changing the spawnrate of the mummy in config to 0. And then changing to peaceful and back to hard fixed it for now.

Trons alternative would work aswell.

**Comment by ItsBlackGear** [comment]
Hello! here's an explanation about what's going on:

mummies are trying to equip a modded armor, and the vanilla renderer cannot identify the correct resource location (that's why it throws an invalid name convention error), so disabling mummies may disable the error until next update which may come soon.

we're already aware of this issue, and we plan on releasing a QoL update to fix this and other issues present in the mod really soon, I'm sorry for the long wait guys.

**Comment by Gbergz** [comment]
Wanna add to this report, seems the config is resetting itself each time the client starts, maybe not resets, but it makes a backup of itself each time.

![bild](https://github.com/teamfusion/rottencreatures/assets/26634554/6d2683a9-7290-4510-8f47-39045b089722)

**Comment by ItsBlackGear** [comment]
yeah, we're also aware of that issue, i managed to create a new config that works properly, will be out next update!

**Comment by ejm039** [comment]
[mummy refined glowstone crash.txt](https://github.com/teamfusion/rottencreatures/files/12115621/mummy.refined.glowstone.crash.txt)
 

This issue still exists. The mummy was wearing refined glowstone armor from mekanism tools.
